docs/docs/features/smart-search.md

```diff
@@ -0,0 +1,37 @@
+# Smart Search
+
+Immich uses Postgres as its search database for both metadata and smart search.
+Smart Search is capable of returning up to 5,000 results for a single search just to avoid browser jank from too many assets being loaded.
```

Comment:
```suggestion
Smart Search is capable of returning up to 5,000 results for a single search (this limit is just to avoid browser jank from too many assets being loaded).
```